LuaUI/widgets/gui_standalone_lobby.lua

```diff
@@ -1,17 +1,20 @@
 function widget:GetInfo()
-  return {
-	name      = "Standalone lobby tools",
-	desc      = "Standalone lobby tools",
-	author    = "gajop",
-	date      = "in the future",
-	license   = "GPL-v2",
-	layer     = 1001,
-	enabled   = true,
-  }
+	return {
+		name      = "Standalone lobby tools",
+		desc      = "Standalone lobby tools",
+		author    = "gajop",
+		date      = "in the future",
+		license   = "GPL-v2",
+		layer     = 1001,
+		enabled   = true,
+	}
 end
 
+spSendCommands = Spring.SendCommands
```

Comment:
You should read about premature optimization, this is a prime example. The impact is minimal. 

I doubt it's actually even an improvement as you're allocating a new variable just for two invocations, which has to be deallocated later. Not to mention you're not making the variable local and thus polluting global scope.

The point is, these miniature optimizations are irrelevant and clear code should always be chosen instead.